spec/features/research_article_reviewer_report_spec.rb

```diff
@@ -84,185 +82,191 @@ def create_reviewer_report_task
     )
   end
 
-  scenario 'A reviewer can fill out their own Reviewer Report, submit it, and see a readonly view of their responses' do
-    create_reviewer_invitation(paper)
-    reviewer_report_task = create_reviewer_report_task
-    report = reviewer_report_task.reviewer_reports.first
-    expect(report.scheduled_events.pluck(:state).uniq).to eq ['active']
-
-    Page.view_paper paper
-    t = paper_page.view_task("Review by #{reviewer.full_name}", ReviewerReportTaskOverlay)
-
-    wait_for_editors # Wait for rich-text editors to instantiate
-
-    t.fill_in_report 'reviewer_report--competing_interests--detail' => 'I have no competing interests'
-    t.submit_report
-    t.confirm_submit_report
-
-    expect(page).to have_selector('.answer-text', text: 'I have no competing interests')
-    expect(report.scheduled_events.pluck(:state).uniq).to eq ['canceled']
-  end
-
-  scenario 'A review can see their previous rounds of review' do
-    # seed the Upload Manuscript card so that it can be created after a decision has been registered
-    ctt = CardTaskType.find_by(task_class: "TahiStandardTasks::UploadManuscriptTask")
-    FactoryGirl.create(:card, :versioned, card_task_type: ctt)
-
-    create_reviewer_invitation(paper)
-    reviewer_report_task = create_reviewer_report_task
+  context 'reviewer is logged in' do
+    before do
+      login_as(reviewer, scope: :user)
+      visit "/"
+    end
```

Comment:
Also assuming this is the only real change here in this file.

Reply:
Correct.

Reply:
Well, there were a few things to make rubocop happy.